guides/manual-installation.md

```diff
@@ -0,0 +1,279 @@
+## Scope
+
+This documents covers the step to manually install trento server without relying on the trento helm chart or kubernetes all together. The lastest
```

Comment:
Should we also mention the ansible option here? 
https://github.com/trento-project/ansible